docs/pages/reference/cli.mdx

```diff
@@ -2373,6 +2396,113 @@ $ tctl alerts create \
     "The system is under maintenance, functionality may be limited."
 ```
 
+### tctl devices add
```

Comment:
Are these entries being added in any sort of structured order? If not, may I suggest listing them alphabetically?

Reply:
The entries are sorted according to `tctl devices --help`, which uses a logical ordering of sorts: device verbs first (add,ls,rm), then enrollment tokens (enroll). Happy to change as well.

Reply:
Following that convention makes sense from that perspective. I'm thinking about the perspective of the user who doesn't have this level of headers listed in our ToC. They may know what they're looking for and be using ctrl/cmd+f to find it, or they may be scrolling the entire document/section to learn. Especially in the latter case, alphabetical ordering can help them keep from losing place.

Reply:
I find this page a bit confusing too, with the lack of navigation and the fact that most of it seems to have no sensible order. Maybe we could add navigation and/or sort the commands consistently throughout? (In a separate PR, that is.)

Sorted `tctl devices` commands alphabetically, but left everything else as-is.

Reply:
Thanks! I spun off an issue to tackle the larger doc's sorting.